### PR TITLE
Log rendering Improvements for beta

### PR DESCRIPTION
**Pull request by abeisgoat**
Adds metadata filtering and other tweaks for beta

### CODE REVIEW

File: src/components/LogViewer/History.tsx [hunk 1]
```diff
@@ -41,6 +41,38 @@ const formatTimestamp = (timestamp: number) => {
     .join(':');
 };
 
+const getFilterTag = (appendToQuery: Function, log: LogEntry) => {
+  const metadata = log.data.metadata;
+  let tagButton;
+
+  console.log(metadata);
```
Comment (tstirrat) [inline src/components/LogViewer/History.tsx:48]: remove console.log

File: src/components/LogViewer/History.tsx [hunk 2]
```diff
@@ -41,6 +41,38 @@ const formatTimestamp = (timestamp: number) => {
     .join(':');
 };
 
+const getFilterTag = (appendToQuery: Function, log: LogEntry) => {
```
Comment (tstirrat) [inline src/components/LogViewer/History.tsx:44]: getFilterTag is really a functional component, how would you feel about:

```js
const FilterTag: React.FC<{...}> = ({appendToQuery, log}) => {
  return <div>...</div>
};
```

and 

```jsx
<FilterTag log={log} appendToQuery={appendToQuery} />
```

File: src/components/LogViewer/index.tsx [hunk 3]
```diff
@@ -64,10 +64,10 @@ export const LogViewer: React.FC<Props> = ({
     const webSocket = new ReconnectingWebSocket(config);
     webSocket.listener = (log: LogEntry) => {
       //todo: remove hack to cut off icon
-      log.message = log.message
-        .split(' ')
-        .slice(2)
-        .join(' ');
+      // log.message = log.message
```
Comment (tstirrat) [inline src/components/LogViewer/index.tsx:67]: not needed anymore? remove dead code

**Comment by abeisgoat** [comment]
Screenshot from 2020-05-14 15-42-25

screenshot for those interested

File: src/components/LogViewer/History.tsx (hunk 2 above)
Comment (abeisgoat) [reply src/components/LogViewer/History.tsx]: smart smart

**Pull request merged**